src/GPL-2.0+.xml

```diff
@@ -0,0 +1,311 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="true" licenseId="GPL-2.0+" isDeprecated="true" deprecatedVersion="2.0rc2"
```

Comment:
I'd rather not use RC versions in `deprecatedVersion`, since those are a lower tier than non-RC versions.  For example, if a license was marked as deprecated in 2.0rc2 but we changed our minds and removed the deprecation by 2.0, I'd consider the license “never deprecated”.  If that argument makes sense, I think we want to use `deprecatedVersion="2.0"` for the licenses where you currently have `2.0rc2`.

Reply:
2.0rc2 was an actual version/release. I can't remember the details of all that now, but we can rewrite history, so this needs to be left as is. :)

Reply:
> 2.0rc2 was an actual version/release. I can't remember the details of all that now, but we can rewrite history, so this needs to be left as is. :)

I think you meant “can't”.  I'm not contesting the existence of the rc2 tag, I just think that tag was a pre-release (i.e. release candidate) and not a release.  [From SemVer][1] (which we don't formally follow, but which serves as a good example of conventional use):

> Pre-release versions have a lower precedence than the associated normal version. A pre-release version indicates that the version is unstable and might not satisfy the intended compatibility requirements as denoted by its associated normal version.

Are you aware of any consumers (e.g. the SPDX spec) that reference the 2.0-rc2 license list?  I'd have expected consumers to wait and reference the normal 2.0 release.

[1]: https://semver.org/spec/v2.0.0.html#spec-item-9